gatorgrouper_django/gatorgrouper/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+
+# Create your models here.
+class Professor(models.Model):
+    email = models.EmailField(max_length = 200)
+    professor_id = models.AutoField(primary_key=True)
+    first_name = models.CharField(max_length = 30)
+    last_name = models.CharField(max_length = 30)
+
+    def __str__(self):
+        return '{}'.format(self.email)
+
+
+class Semester_Class(models.Model):
+    professor_id = models.ForeignKey(
+            Professor,
+            on_delete=models.CASCADE,
+            )
+    semester = models.CharField(max_length = 50)
```

Comment:
Do we need to check that semester strings have a consistent format before saving to the DB? Or will it be okay if some semester strings have different formats?

Reply:
I was thinking of changing this and making semesters their own db. That way there's a more consistent way of saving them into the DB. Idk though

Reply:
Django uses a single DB that contains different tables using different schema.

The format of the data that you enter can be specified in the model using, in this case, CharField and EmailField, but you can use any of the other types supported by Django and it will make sure the data is of that type before allowing you to continue.  It will not allow you to enter something that is the wrong type or format.  However, seeing as there will be users from different institutions not having a strict check on the data might be the best route.

Reply:
> Hello @barrezuetai, I am wondering whether or not these changes will ultimately prevent GatorGrouper from also being used as a command-line application. My hope is that it will be possible to run it in the command-line while also interacting with it through a Django-based web interface. Is that also your plan?

I didn't realize that we still wanted to keep it a command-line application along with a web app. In that case, it would change it so that it can't be run from the command-line. I'm not sure how we can make it both besides copying the code and placing one outside the Django directory and keeping another copy inside...

Reply:
@barrezuetai Using Field.choices might work? https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.Field.choices

Reply:
@aubreypc I forgot about those! They'll definitely make things easier. I'll integrate them into some of the attributes tonight 💯

Reply:
Added choices (Spring, 2019 and Fall, 2019) to the Semester_Class class to standardize responses.